app/Resources/views/composer/step2.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends "composer/step_layout.html.twig" %}
+
+{% block step_title %}
+    Step 2: Install the database
+{% endblock %}
+
+{% block step_description %}
+    You need a MySQL server to store the database.<br/>
+    We are going to create the Database and import the schema and the first clean dataset.
+{% endblock %}
+
+
+{% block step_contnet %}
+    <div class="help">
+        <p>If you do not have a MySQL server ready, you can create one quickly using Docker.</p>
+    </div>
+    <div class="orange">
+        <p>&#36; docker run --rm -p 3333:3306 --name ezdbcontainer -e MYSQL_ROOT_PASSWORD=ezplatform mariadb:5.5</p>
+    </div>
+
+    <div class="help">
+        <p># Create the database</p>
+    </div>
+    <div class="orange">
+        <p>&#36; mysql -u root -p -P 3333 -h 127.0.0.1 -e "CREATE DATABASE ezplatform CHARACTER SET utf8;"</p>
+    </div>
```

Comment:
missing help text describing what password to use when it prompts for it, or inline password in command above.